app/pods/publications/publication/documents/template.hbs

```diff
@@ -46,8 +40,157 @@
     </WebComponents::AuToolbar::Group>
   </WebComponents::AuToolbar>
 </div>
+
+
+{{!-- TABLE FOR DOCUMENTS --}}
+
+
 <div class="u-mb-3">
-  <WebComponents::AuAlert @message={{t "no-documents-yet"}}></WebComponents::AuAlert>
+  {{#if (gt (await @model.pieces.length) 0)}}
+    <table class="auk-table">
+      <thead>
+        <tr>
+          <th></th>
+          <th class="auk-table__col--6">{{t "document"}}</th>
+          <th class="u-text-nowrap">{{t "uploaded-at"}}</th>
+          <th colspan="2">{{t "translations"}}</th>
+          <th colspan="2">{{t "signatures"}}</th>
+          <th colspan="2">{{t "proof"}}</th>
+          <th></th>
+        </tr>
+        <tr>
+          <th></th>
+          <th></th>
+          <th></th>
+          <th>{{t "needed?"}}</th>
+          <th>{{t "present?"}}</th>
+          <th>{{t "needed?"}}</th>
+          <th>{{t "present?"}}</th>
+          <th>{{t "needed?"}}</th>
+          <th>{{t "present?"}}</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each @model.pieces as |piece|}}
+          <tr>
+            <td>
+              <WebComponents::AuButton 
+                {{on "click" (fn this.toggleFolderCollapse piece)}} 
```

Comment:
Misschien de moete om eens te experimenteren met de [let helper](https://api.emberjs.com/ember/3.22/classes/Ember.Templates.helpers/methods/let?anchor=let) hier (een hash toekennen aan `row` met property "collapsed"=true). Ik weet niet zeker of het lukt om die hash values dan aan te passen via een action, maar het proberen waard. Als we de huidige approach op meerdere plaatsen gaan gebruiken, gaan we snel rare dingen zien. Een model is eigenlijk geen plaats om interface state op bij te houden ...